src/utils/base64-utils.ts

```
@@ -0,0 +1,29 @@
+
+    return bytes.buffer;
+  }
+
+  public static stringToBase64Url(str: string): string {
+    return this.arrayBufferToBase64Url(new TextEncoder().encode(str).buffer);
+  }
+
+  public static base64UrlToString(base64url: string): string {
+    return new TextDecoder().decode(this.base64UrlToArrayBuffer(base64url));
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)